### PR TITLE
Rwc2018/challenge eegpsr2

### PR DESCRIPTION
**Pull request by MatthijsBurgh**

### CODE REVIEW

File: challenge_eegpsr/data/EEGPSR Cat4 Examples/EEGPSR Cat4 Examples.txt [hunk 1]
```diff
@@ -0,0 +1,11459 @@
+﻿###############################################################################
```
Comment (MatthijsBurgh) [inline challenge_eegpsr/data/EEGPSR Cat4 Examples/EEGPSR Cat4 Examples.txt]: Please remove this file

File: challenge_eegpsr/data/from-official-command-generator.txt [hunk 2]
```diff
@@ -1,154 +1,386 @@
-Navigate to the cabinet, locate Jacob, grasp the apple from the sideshelf, and deliver it on the
-TV stand.
+##############################################################################
```
Comment (MatthijsBurgh) [inline challenge_eegpsr/data/from-official-command-generator.txt]: Please remove this file

File: challenge_eegpsr/data/unique-commands-first-type.txt [hunk 3]
```diff
@@ -0,0 +1,25 @@
+and answer a question
```
Comment (MatthijsBurgh) [inline challenge_eegpsr/data/unique-commands-first-type.txt]: Please remove this file

File: challenge_eegpsr/data/unique-commands-second-type.txt [hunk 4]
```diff
@@ -0,0 +1,46 @@
+﻿and and answer a question
```
Comment (MatthijsBurgh) [inline challenge_eegpsr/data/unique-commands-second-type.txt]: Please remove this file

File: challenge_eegpsr/data/unique-commands-third-type.txt [hunk 5]
```diff
@@ -0,0 +1,44 @@
+﻿and bring it to John at the furniture
```
Comment (MatthijsBurgh) [inline challenge_eegpsr/data/unique-commands-third-type.txt]: Please remove this file

**Comment by MatthijsBurgh** [comment]
ping ;)

**Comment by LoyVanBeek** [comment]
@MatthijsBurgh ther'e a button to approve stuff :-)

**Comment by MatthijsBurgh** [comment]
I created the PR, so I can’t approve

**Pull request merged**